add-on/background.js

```diff
@@ -116,19 +116,32 @@ function youTubeMusicScriptThatClicksOn (actionName) {
 
 function podcastsScriptThatClicksOn(actionName) {
   const script = function (actionName) {
-
-    const audioNode = document.querySelector("audio");
-    switch (actionName) {
-      case 'rewind':
-        return audioNode.currentTime -= 10
-      case 'forward':
-        return audioNode.currentTime += 30
-      case 'play-pause':
-        if (audioNode.paused)
-          return audioNode.play()
-        else
-          return audioNode.pause()
+    const playButtonsContainer = document.querySelector("#ZCHFDb > div > div > c-wiz > div.d5Ncgc > div.dq2Yed > div.yGpUe > div.Jy7gWd");
+    
+    if (playButtonsContainer != null) {
+      switch (actionName) {
+        case 'rewind':
+          return playButtonsContainer.children[0].children[0].click();
+        case 'forward':
+          return playButtonsContainer.children[2].children[0].click();
+        case 'play-pause':
+            return playButtonsContainer.children[1].children[0].click();
+      }
+    } else {
+      const audioNode = document.querySelector("audio");
+      switch (actionName) {
+        case 'rewind':
+          return audioNode.currentTime -= 10
+        case 'forward':
+          return audioNode.currentTime += 30
+        case 'play-pause':
+          if (audioNode.paused)
+            return audioNode.play()
+          else
+            return audioNode.pause()
```

Comment:
Yeah, those minified class names will stop working pretty fast. I agree with you, no point of doing this.
The only reliable hint I was able to find was the accessibility labels for Aria. Those will work only for English locale, but it is best we can do.

Can you refactor this, so the fallback happens for each button?

It should try to find a button using aria hint, if the button is missing, then  use the `audioNode` as fallback.

Below  are aria-based lookups for buttons, you can `.click()` on each of them:

- `rewind` – `document.body.querySelector('div[aria-label="Rewind 10 seconds"]')` 
- `forward` – `document.body.querySelector('div[aria-label="Fast forward 30 seconds"]')`
- `play-pause`– this one is tricky because aria has separate labels for each state, so you need to look for both:
  play – `document.body.querySelector('div[aria-label="Play"]')`
  pause – `document.body.querySelector('div[aria-label="Pause"]')`